datafusion/expr/Cargo.toml

```diff
@@ -43,6 +43,7 @@ arrow-array = { workspace = true }
 chrono = { workspace = true }
 datafusion-common = { workspace = true, default-features = true }
 paste = "^1.0"
+serde_json = { workspace = true }
```

Comment:
Maybe we could simply make the JSON directly with string manipulation... Maybe it doesn't matter

Reply:
To be honest I'm not a big fan of manipulating string directly since it's not easy to maintain. As you said, `serde_json` is already used by many places, so I think it fine to use it now?

Reply:
Yeah I think it is fine. I am just trying to keep the dependency chain for datafusion as minimal as posisble (to support wasm, etc).

It seems like we always add dependencies but never remove them 🤷

Reply:
There is a `udeps` which could figure out unused crates, but it requires unstable rust.

Reply:
Tracked here: https://github.com/apache/arrow-datafusion/issues/9881